tasks/package.yml

```diff
@@ -11,7 +11,7 @@
 
 - name: Nginx | Make sure nginx is installed (package)
   apt:
-    pkg: nginx
```

Comment:
I think this was the purpose of the `nginx` var .. perhaps you could just just wrap the existing value in template tags e.g `pkg: "{{ nginx }}"`` see https://github.com/ANXS/nginx/pull/75/files for clarification on original intent .. AFAICT it wont intefer with tengine, because tengine has to be built from source..